python/hail/expr/aggregators.py

```diff
@@ -814,8 +814,8 @@ def inbreeding(expr, prior):
     return construct_expr(ast, t, Indices(source=indices.source), aggregations.push(Aggregation(indices, refs)), joins)
 
 
-@typecheck(expr=oneof(Aggregable, expr_call), variant=expr_variant)
-def call_stats(expr, variant):
+@typecheck(expr=oneof(Aggregable, expr_call), alleles=expr_list)
```

Comment:
Shouldn't alleles be an ArrayStringExpression?  Just for my own purposes, I think I'm confused about how typechecking of exprs is supposed to work.

Reply:
no, because you should be able to put in `['A', 'T']` if you want. That won't typecheck as ArrayStringExpression, but will get converted with expr_list. We need to add better expr typecheckers that take a type.